components/builder-scheduler/src/data_store.rs

```diff
@@ -347,9 +347,9 @@ impl DataStore {
 
         // Check for an active (Pending or Dispatched) group with a given root project name
         migrator.migrate("scheduler",
-                        r#"CREATE OR REPLACE FUNCTION check_active_group_v1(project_name text) RETURNS SETOF groups AS $$
+                        r#"CREATE OR REPLACE FUNCTION check_active_group_v1(pname text) RETURNS SETOF groups AS $$
```

Comment:
Oooof... this has bitten me so many times in the past.

A pattern I have adopted is to always prefix my function parameters with `p_` (which I never use for column names). For PL/pgSQL functions with internal variables, I'll similarly use a prefix of `v_`. You avoid name clashes like this, and can pretty quickly determine where each relevant bit of data comes from. Kind of like a limited Hungarian notation style for stored procedures 😄